bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -135,6 +126,17 @@ class AudioManager {
     this.isConnecting = false;
     this.isConnected = true;
 
+    // listen to the VoiceUsers changes and update the flag
+    if(!this.muteHandle) {
```

Comment:
Could you initialize this variable in the constructor?
Just so that we knew about its existence.

Reply:
Sure. Good point.